modules/articles/tests/server/admin.article.server.routes.tests.js

```diff
@@ -23,7 +23,7 @@ var app,
 describe('Article Admin CRUD tests', function () {
   before(function (done) {
     // Get application
-    app = express.init(mongoose);
+    app = express.init(mongoose.connection.db);
```

Comment:
Yep. And indeed the returned value of the mongoose.js line 27 was just the db object being resolved with.